xbmc/filesystem/ISO9660File.cpp

```diff
@@ -23,18 +23,19 @@ bool CISO9660File::Open(const CURL& url)
   if (m_iso && m_stat)
     return true;
 
-  if (!m_iso->open(url.GetHostName().c_str()))
+  if (!m_iso->open_fuzzy(url.GetHostName().c_str()))
```

Comment:
perhaps a fallback to `open_fuzzy` if `open` fails may be safer?

Reply:
Sounds good

Reply:
not sure if `m_iso->open()` really fails when we're facing the issue. actually runtime testing without a fallback.